api/src/pcapi/core/external/attributes/api.py

```diff
@@ -111,6 +111,14 @@ def get_pro_attributes(email: str) -> models.ProAttributes:
             )
             .joinedload(offerers_models.Venue.venueLabel)
             .load_only(offerers_models.VenueLabel.label),
+            joinedload(users_models.User.UserOfferers)
+            .joinedload(offerers_models.UserOfferer.offerer)
+            .joinedload(offerers_models.Offerer.managedVenues)
+            .joinedload(offerers_models.Venue.collectiveOffers),
```

Comment:
Je ne sais pas combien d'offres collectives on a pour les plus gros lieux (ne pas l'indiquer ici sur GitHub, ça n'a pas de raison d'être public), mais est-ce que ça n'est pas un peu coûteux de les récupérer tous (et de les stocker en mémoire, et idem pour les Stock et Booking dont on a aussi besoin, cf. mon commentaire plus bas) comme ça, alors que finalement on a simplement besoin de savoir s'il existe des offres collectives qui répondent au critère demandé ? Peut-être que faire une requête SQL unique pour chaque structure : `SELECT 1 WHERE EXISTS (select id from collective_offer join venue on ... join offerer on ... where offerer.id in ... and collective_offer.isActive and ...)` serait plus raisonnable ?